zipline/utils/calendars/trading_calendar.py

```diff
@@ -690,11 +699,19 @@ def minute_to_session_label(self, dt, direction="next"):
         pd.Timestamp (midnight UTC)
             The label of the containing session.
         """
+        if direction == "next":
```

Comment:
How does this compare to just decorating this method with `cache_tools.lru_cache(maxsize=1)`?